packages/shared-component--statement/dist/statement.html

```diff
@@ -0,0 +1,61 @@
+<div class="coop-c-statement-block coop-c-statement-block--teal">
```

Comment:
Wondering if this is a <blockquote> or an <aside> or do we lack that context without it being in a page?